src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#image">
+        <img
+        class="image"
+        src="images\moyo_1.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item li_is_active">
+            <a href="#Apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#Samsung" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#Smartphones" class="nav-link">Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#Laptops"
+              class="nav-link laptops-link"
+              data-qa="hover"
+            >
+            Laptops & Computers
```

Comment:
```suggestion
              Laptops & Computers
```

Text inside tag should be with two more spaces then opening tag if it in separate line